shopyo/shopyoapi/utils.py

```diff
@@ -0,0 +1,25 @@
+import shutil
+
+
+def trycopytree(source, dest):
+    try:
+        shutil.copytree(source, dest)
+        print("done copying {} to {}".format(source, dest))
+    except Exception as e:
+        print(e)
+
+
+def trycopy(source, dest):
+    try:
+        shutil.copy(source, dest)
+        print("done copying {} to {}".format(source, dest))
+    except Exception as e:
```

Comment:
@Abdur-rahmaanJ Hey is there a particular exception you are expecting here?

Reply:
Thought .copy would throw up errors if file exists but they do not, must maybe remove it

Reply:
Got it.